src/main/java/com/sublinks/sublinksapi/api/lemmy/v3/user/controllers/UserController.java

```diff
@@ -491,9 +538,8 @@ ExportSettingsResponse export_settings(final JwtPerson principal) {
     // @todo Add Blocklist for communities, User and Instnaces
 
     List<String> blocked_community = new ArrayList<>();
-    linkPersonCommunityService.getPersonLinkByType(person, LinkPersonCommunityType.blocked)
-        .forEach(
-            linkPersonCommunity -> blocked_community.add(linkPersonCommunity.getActivityPubId()));
+    linkPersonCommunityService.getPersonLinkByType(person, LinkPersonCommunityType.blocked).forEach(
+        linkPersonCommunity -> blocked_community.add(linkPersonCommunity.getActivityPubId()));
```

Comment:
Just like the other formatting changes, this feels less readable than the original

Reply:
I dont have an idea why does .editorconfig enforce it then?